### PR TITLE
Phone Verification: Use English for India Phone Numbers

### LINKED ISSUE #1124: Change default language for SMS verification in India
The Twilio API guesses a language for SMS verification based on the country of the phone number. We got a report in Discord that for India it defaults to Hindi which a large part of the population cannot read so it'd be better if we default to English for India. We can explicitly set the language to English for Indian phone numbers.

screen shot 2018-12-22 at 11 35 01 am

### PR DESCRIPTION
**Pull request by BrunoBernardino**
### Description:

Fixes #1124.

Hindi is Twilio's default for Indian phone numbers, but according to a user report (and after some quick investigation), English appears to be preferred.

This includes a regression test.

I'd love to know what should I do to get this merged.

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [x] Wrap any displayed ETH addresses with [`formattedAddress`](https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/utils/user.js#L15-L17)
- [x] Wrap any new text/strings for translation
- [x] Map any new environment variables with a default value in the Webpack config
- [x] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/origin-docs)

### CODE REVIEW

File: origin-bridge/logic/attestation_service.py [hunk 1]
```diff
@@ -81,6 +81,10 @@ def send_phone_verification(country_calling_code, phone, method, locale):
             # If a locale is not set Twilio will use a sensible default based on
             # the country of the telephone number
             params['locale'] = locale
+        elif country_calling_code == '91':
+            # Locale not provided, and calling India, so we want to set English
+            # instead of Hindi, which is Twilio's default there (#1124)
```
Comment (BrunoBernardino) [inline origin-bridge/logic/attestation_service.py]: I haven't seen this pattern here of referencing issues, not sure if it's acceptable.

File: origin-bridge/tests/services/test_attestation.py [hunk 2]
```diff
@@ -194,6 +194,28 @@ def test_verify_phone_invalid_code():
     assert(validation_err.value.field_names[0]) == 'code'
 
 
+@mock.patch('logic.attestation_service.requests')
+@responses.activate
```
Comment (BrunoBernardino) [inline origin-bridge/tests/services/test_attestation.py]: I didn't see in other tests a way to investigate the request via `responses`, but if there's one I'd love to learn about it as it's probably better than mocking the request.

File: origin-bridge/tests/services/test_attestation.py [hunk 3]
```diff
@@ -194,6 +194,28 @@ def test_verify_phone_invalid_code():
...
+    args = {
+        'country_calling_code': '91',
+        'phone': '12341234',
+        'method': 'sms',
+        'locale': None
+    }
+    response = VerificationService.send_phone_verification(**args)
+
+    assert len(mock_requests.post.call_args_list) == 1
+    assert mock_requests.post.call_args_list[0][1]['params']['locale'] == 'en'
```
Comment (BrunoBernardino) [inline origin-bridge/tests/services/test_attestation.py]: I didn't use `.get()` here as if we don't get these dict keys it's ok to fail the test.

**Comment by micahalcorn** [comment]
Thanks @BrunoBernardino 💙 I've added you to the GitHub organization so that you can push a branch directly to the repo in the future instead of forking. I'll ask @tomlinton to review this.

**Comment by tomlinton** [comment]
This looks great @BrunoBernardino, thanks! I'll test it out later today then merge it if everything looks good.

File: origin-bridge/logic/attestation_service.py (hunk 1 above)
Comment (tomlinton) [reply origin-bridge/logic/attestation_service.py]: It's something I think we should do more of 👍

File: origin-bridge/tests/services/test_attestation.py (hunk 2 above)
Comment (tomlinton) [reply origin-bridge/tests/services/test_attestation.py]: You should be able to do something like `responses.calls[0].request` to get the original request. This is fine, but feel free to refine it if you want.

**Pull request merged**